Wox.Core/Resource/Theme.cs

```diff
@@ -76,15 +76,11 @@ public bool ChangeTheme(string theme)
                 Settings.Theme = theme;
 
                 var dicts = Application.Current.Resources.MergedDictionaries;
-                //always allow re-loading default theme, in case of failure of switching to a new theme from default theme
-                if (_oldTheme != theme || theme == defaultTheme)
```

Comment:
why remove this check?
what will happen "in case of failure of switching to a new theme from default theme"?

Reply:
remove because theme needs to reload to change fonts
this only allowed to reload default theme, which means to see the font change you needed to change the theme

Reply:
fail code is below, the exception handler